apiserver/facades/controller/crossmodelsecrets/register.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package crossmodelsecrets
+
+import (
+	"reflect"
+
+	"github.com/juju/errors"
+
+	"github.com/juju/juju/apiserver/common/secrets"
+	"github.com/juju/juju/apiserver/facade"
+	"github.com/juju/juju/secrets/provider"
+	"github.com/juju/juju/state"
+)
+
+// Register is called to expose a package of facades onto a given registry.
+func Register(registry facade.FacadeRegistry) {
+	registry.MustRegister("CrossModelSecrets", 1, func(ctx facade.Context) (facade.Facade, error) {
+		return newStateCrossModelSecretsAPI(ctx)
+	}, reflect.TypeOf((*CrossModelSecretsAPI)(nil)))
+}
+
+// newStateCrossModelSecretsAPI creates a new server-side CrossModelSecrets API facade
+// backed by global state.
+func newStateCrossModelSecretsAPI(ctx facade.Context) (*CrossModelSecretsAPI, error) {
+	// TODO(cmr secrets)
+	// authCtxt := ctx.Resources().Get("secretAccessAuthContext").(common.ValueResource).Value
```

Comment:
remove?

Reply:
This is being done in the next PR so is left here as a placeholder in this PR